test/Formik.test.tsx

```diff
@@ -44,29 +45,36 @@ function Form({
   );
 }
 
-const InitialValues: Values = { name: 'jared' };
-
-function renderFormik<V>(props?: Partial<FormikConfig<V>>) {
+const InitialValues: Values = {
```

Comment:
I like naming constants in a way they resemble constant :) E.g. `INITIAL_VALUES`